src/options.rs

```diff
@@ -319,7 +319,7 @@ pub enum Command {
     #[edb(inherit(CloudOptions), hide=true)]
     Cloud(CloudCommand),
     /// EdgeDB Cloud authentication
```

Comment:
The doc and the inheritance of `CloudOptions` have probably been copy-pasted by mistake?

Reply:
Right. Fixed that.